nixio/property.py

```diff
@@ -215,9 +227,21 @@ def odml_type(self, new_type):
 
         self._h5dataset.set_attr("odml_type", str(new_type))
 
+    def _read_old_values(self):
+        val = self._h5dataset.dataset[:]
+        val_tu = tuple()
+        for v in val:
+            v = v["value"]
+            val_tu += (v,)
+        return val_tu
```

Comment:
@jgrewe @gicmo What's the current situation in NIX with reading old-style values and is there any plan to change it?  Do we ignore the extra information and just return a _new style_ tuple of raw values?

Reply:
at the moment we just do that, returning a vector of Variant, i.e. the raw values, reference was swept under the rug, methinks...
need to remedy that

Reply:
We can go ahead and merge then and see how it develops in NIX.